src/PIL/Image.py

```diff
@@ -2143,14 +2146,17 @@ def thumbnail(self, size, resample=BICUBIC):
             x = int(max(x * size[1] / y, 1))
             y = int(size[1])
         size = x, y
+        box = None
 
         if size == self.size:
             return
 
-        self.draft(None, size)
+        res = self.draft(None, size)
+        if res is not None:
+            box = res[1]
 
         if self.size != size:
-            im = self.resize(size, resample)
+            im = self.resize(size, resample, box=box)
```

Comment:
```suggestion
            im = self.resize(size, resample, box)
```
I suggest not specifying the argument name - it's not necessary, and the variable name already clearly indicates the purpose.

Reply:
In truth, I think this is a case where we could use [Keyword-Only Arguments](https://www.python.org/dev/peps/pep-3102/).

It seems obvious that we can omit the argument name when you watch on it and so you know that it exists and what exactly it is doing. But once you omit it, you can't know exactly where your variable goes.